src/main/java/uz/ccrew/matchmaking/service/impl/MatchServiceImpl.java

```diff
@@ -121,4 +123,26 @@ private void checkMatchToPrepare(Match match, MatchMode mode, TeamType teamType)
         match.setStatus(MatchStatus.PREPARED);
         matchRepository.save(match);
     }
+
+    public void handleResult(String matchId,String teamID){
+        UUID matchUUID = UUID.fromString(matchId);
```

Comment:
Match qaysi serverda bogan bosa osha serverni is busy fieldini false qilish kere